design-system/materials/internals/definition.yaml

```diff
@@ -1013,6 +1015,8 @@ decisionGroupsByTheme:
           choice: 'inherit'
         padding-for-money-input-currency-dropdown:
           choice: '0 var(--spacing-20)'
+        padding-for-selectable-search-input-dropdown:
+          choice: '0 var(--spacing-20)'
```

Comment:
Do we need a design token for this? 🤔

Reply:
It may not be needed. What do you think @CarlosCortizasCT

Reply:
Seems like the token above. Maybe follow up and remove both if that's decided.

Reply:
If we're having the same value for both themes, I would consider not adding a token for it.

As for the token above, it does not have the same value for both themes.

Reply:
Thanks, Updated here [d1e1958](https://github.com/commercetools/ui-kit/pull/2426/commits/d1e1958c7c0c1bb0ddf1fd144c80cae646684d0f)